browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -528,10 +536,6 @@ void RewardsDOMHandler::GetRewardsParameters(const base::ListValue* args) {
 
 void RewardsDOMHandler::OnGetRewardsParameters(
     ledger::type::RewardsParametersPtr parameters) {
-  if (!web_ui()->CanCallJavascript()) {
```

Comment:
Yeah, I think we should keep these checks because AFAIK the rewards service could in principle run these callbacks while the message handler is in a "JS disallowed" state.

Reply:
The checks have been added back.